.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.22.2
```

Comment:
Please use a major go version, like `1.22`, without the patch, for the CI

Reply:
[b9268a0](https://github.com/gnolang/blog/pull/77/commits/b9268a0f849d3b715bcd60193b049016b0fa702e)